### ISSUE TITLE
have resnet50 for imagenet weight?

### ISSUE THREAD

**Issue by Phoebe-star** [piyush2896/ResNet50-Tensorflow#1]
you have resnet50 for imagenet weight to test the net?
thanks

**Comment by Phoebe-star** [comment]
and can you provide yuour dataset and training request (like labels) , 
if you can not provide  total dataset , just a little  dataset 
thanks , I just want to run it
have you pretrain model?
